conf/modules/annotate_genome_snvs.config

```diff
@@ -0,0 +1,120 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args            = Additional arguments appended to command in module.
+        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix          = File name prefix for output files.
+        ext.when            = Conditional clause
+----------------------------------------------------------------------------------------
+*/
+
+//
+// SNV annotation options
+//
+
+process {
+
+    if (!params.skip_snv_annotation) {
+        withName: '.*ANNOTATE_GENOME_SNVS:BCFTOOLS_ROH' {
+            ext.args = { "--samples ${meta.probands.unique().first()} --skip-indels " }
+            ext.prefix = { "${meta.id}_roh" }
+        }
+
+        withName: '.*ANNOTATE_GENOME_SNVS:RHOCALL_ANNOTATE' {
+            ext.args = { "--v14 " }
+        }
+
+        withName: '.*ANNOTATE_GENOME_SNVS:ZIP_TABIX_ROHCALL' {
+            ext.prefix = { "${input.simpleName}" }
+        }
+
+        withName: '.*ANNOTATE_GENOME_SNVS:VCFANNO' {
+            ext.prefix = { "${vcf.simpleName}_vcfanno" }
+        }
+
+        withName: '.*ANNOTATE_GENOME_SNVS:UPD_SITES' {
+            ext.prefix = { "${vcf.simpleName}_upd_sites" }
+            ext.args = {"--af-tag GNOMADAF --proband ${meta.upd_child} --mother ${meta.mother} --father ${meta.father} sites"}
+        }
+
+        withName: '.*ANNOTATE_GENOME_SNVS:UPD_REGIONS' {
+            ext.prefix = { "${vcf.simpleName}_upd_regions" }
+            ext.args = {"--af-tag GNOMADAF --proband ${meta.upd_child} --mother ${meta.mother} --father ${meta.father} regions --min-size 5 --min-sites 1"}
+            ext.when = { !(workflow.profile.tokenize(',').intersect(['test', 'test_one_sample']).size() >= 1) || workflow.stubRun }
+        }
+
+        withName: '.*ANNOTATE_GENOME_SNVS:CHROMOGRAPH_SITES' {
+            ext.prefix = { "${sites.simpleName}_chromograph" }
+            ext.args = { "--euploid" }
+            tag =  {"${meta7.id}"}
+            publishDir = [
+                path: { "${params.outdir}/annotate_snv/genome" },
+                mode: params.publish_dir_mode,
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            ]
+        }
+
+        withName: '.*ANNOTATE_GENOME_SNVS:CHROMOGRAPH_REGIONS' {
+            ext.prefix = { "${regions.simpleName}_chromograph" }
+            ext.args = { '--euploid' }
+            ext.when = { !(workflow.profile.tokenize(',').intersect(['test', 'test_one_sample', 'test_sentieon']).size() >= 1) || workflow.stubRun }
```

Comment:
Is it possible to run more than one test-profile at a time? (In Sarek, that is not possible.)

Reply:
Not possible here either.